Cargo.toml

```diff
@@ -12,13 +12,13 @@ readme = "README.md"
 [dependencies]
 byteorder = "1"
 flate2 = { version = "1", features = ["rust_backend"], default-features = false }
-lzxd = "0.2"
+lzxd = "0.2.5"
```

Comment:
For more details: https://github.com/tokio-rs/axum/pull/2346#issuecomment-1825633264

Reply:
This patch version has some fixes.